src/block/Crops.php

```diff
@@ -111,4 +112,11 @@ public function onRandomTick() : void{
 			}
 		}
 	}
-}
+
+	public function onEntityFall(Entity $entity, float $fallDistance, Vector3 $fallPos) : void{
+		$blockBelow = $this->getSide(Facing::DOWN);
+		if($blockBelow->getId() === BlockLegacyIds::FARMLAND){
+			$blockBelow->onEntityFall($entity, $fallDistance, $fallPos);
```

Comment:
I don't like this. It feels too much like a hack.

Reply:
wdym? fall() in Living returns the block the player falls onto, including crops